test/spec/bulk-memory-operations/binary.txt

```diff
@@ -89,15 +89,16 @@ out/test/spec/bulk-memory-operations/binary.wast:272: assert_malformed passed:
 out/test/spec/bulk-memory-operations/binary.wast:282: assert_malformed passed:
   000000e: error: unable to read i64 leb128: init_expr i64.const value
 out/test/spec/bulk-memory-operations/binary.wast:295: assert_malformed passed:
-  0000022: error: call_indirect reserved value must be 0
+  0000000: error: table variable out of range: 1 (max 1)
```

Comment:
I don't know why the output changes. Still all the tests pass, but it is suspicious. Please double check, this is unknown territory for me.